test/index.html

```diff
@@ -49,79 +49,146 @@
 
-  <h2>Evidence search</h2>
-  <iframe src="./example.evidence-search.html"></iframe>
-  <iframe src="./example.evidence-search.html"></iframe>
+    .show {display:block;}
+    .hidden{display: none;}
+    #dropdownList{
+      height:  300px;
+      overflow: scroll;
+      overflow-x: hidden;
+    }
+    form{
+      display: inline;
+    }
 
-  <hr/>
 
-  <h2>BNF microsite</h2>
-  <iframe src="./example.bnf.html" class="microsite"></iframe>
-  <iframe src="./example.bnf.html"></iframe>
+  </style>
+  <script
+  src="https://code.jquery.com/jquery-3.1.1.js"
+  integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA="
+  crossorigin="anonymous"></script>
+  <script>
+    /* When the user clicks on the button, 
+    toggle between hiding and showing the dropdown content */
+    
+
+    function triggerDropdown() {
+      $("#dropdownList").toggleClass("show");
+    }
 
-  <hr/>
+// Close the dropdown if the user clicks outside of it
+$(window).click(function() {
+  if (!event.target.matches(".dropbtn")) {
 
-  <h2>BNFC microsite</h2>
-  <iframe src="./example.bnfc.html" class="microsite"></iframe>
-  <iframe src="./example.bnfc.html"></iframe>
+  $(".dropdownContent").each(function() {
+    var openDropdown = $(this);
+    if (openDropdown.hasClass("show")) {
+      openDropdown.removeClass("show");
+    }
+  });
+}});
 
-  <hr/>
+$(document).ready(function() {
 
-  <h2>CKS microsite</h2>
-  <iframe src="./example.cks.html" class="microsite"></iframe>
-  <iframe src="./example.cks.html"></iframe>
+ $('.dropdownContent a').click(function(e) {
+  e.preventDefault();
+  var url = $(this).attr("href");
+  var title = $(this).text();
+  renderIframe(url, title);
+});
 
-  <hr/>
 
-  <h2>Appraisals tool</h2>
-  <iframe src="./example.appraisals.html"></iframe>
-  <iframe src="./example.appraisals.html"></iframe>
+ $(".dropbtn").click(function() {
+  triggerDropdown();
+});
 
-  <hr/>
 
-  <h2>ROI tools</h2>
-  <iframe src="./example.roi.html" class="microsite"></iframe>
-  <iframe src="./example.roi.html"></iframe>
+ function renderIframe(url, title) {
+  var $views = $("#views");
+  if($("#compareTopHats").is(":checked")) {
+  $views.append('<hr>');
+ }
+ else {
+   $views.empty();
+  }
+   $views.append('<h1>' + title + '</h1>');
+   $views.append('<iframe src="' + url + '"></iframe>');
+   $views.append('<iframe class="mobile" src="' + url + '"></iframe>');
+}
 
-  <hr/>
 
-  <h2>InDev tool</h2>
-  <iframe src="./example.indev.html"></iframe>
-  <iframe src="./example.indev.html"></iframe>
+});
 
-  <hr/>
+</script>
+</head>
 
-  <h2>Pathways</h2>
-  <iframe src="./example.pathways.html"></iframe>
-  <iframe src="./example.pathways.html"></iframe>
+<body>
 
+  <section>
+
+    <div class="dropdown">
+
+    <button type="menu" class="dropbtn">Project tophats ⮟ </button>
+      <div id="dropdownList"  class="dropdownContent">
+        <a class="button" href="./example.niceorg.html">Nice org</a>
```

Comment:
You can remove the button class from these now that your selector has changed to ".dropdownContent a"